go/tools/gazelle/rules/generator.go

```diff
@@ -91,15 +91,25 @@ func (g *generator) Generate(rel string, pkg *build.Package) ([]*bzl.Rule, error
 		rules = append(rules, p)
 	}
 
-	libName := defaultLibName
-	r, err := g.generateLib(rel, libName, pkg)
+	cgoLibrary := ""
+	if len(pkg.CgoFiles) != 0 || len(pkg.CFiles) != 0 || len(pkg.HFiles) != 0 {
+		cgoLibrary = "cgo_default_library"
```

Comment:
move "cgo_default_library" to be a file-level constant next to the others for clarity.